crypto/camellia/camellia.c

```diff
@@ -57,15 +57,16 @@
 #  define GETU32(p)   SWAP(*((u32 *)(p)))
 #  define PUTU32(p,v) (*((u32 *)(p)) = SWAP((v)))
 # elif defined(__GNUC__) && __GNUC__>=2
+   typedef u32 u32_a1 __attribute((__aligned__(1)));
```

Comment:
Just like with e.g. cbc128 module, it's only appropriate if the definition is conditional on whether or not platform can handle misaligned access. Two options, a) move it into CPU-specific clauses; b) remove whole section. I suggest b).